modules/release-notes/ref-release-notes-fixed-issues.adoc

```diff
@@ -89,4 +89,16 @@ With this update, the Profile dropdown now correctly displays the user&#39;s nam
 * link:https://issues.redhat.com/browse/RHIDP-7380[RHIDP-7380]
 
 
+[id="bug-fix-rhidp-7529"]
+=== Fixed incorrect branding in 1.6.0 Helm chart release
+
+Previously, the upstream branding was used instead of Red Hat branding when releasing the 1.6.0 Helm chart.

```

Comment:
Consider using attributes for versions as well.
```suggestion
Previously, the upstream branding was used instead of {company-name} branding when releasing the 1.6.0 Helm chart.

```